operator/pkg/controllers/master/kubecontrollermanager.go

```diff
@@ -267,34 +265,33 @@ data:
 
 var (
 	disabledFlagsForKube122 = map[string]struct{}{"--horizontal-pod-autoscaler-use-rest-clients": {}}
```

Comment:
`--horizontal-pod-autoscaler-use-rest-clients` we skip this flag for 1.22 and 1.26+. What about 1.23 - 1.25?
If that's something missing, can you please add it. Please feel free to address this in a followup PR given this PR is targeting 1.26 and 1.27.

Reply:
For 1.23 - 1.25, we use the same variable that is named disabledFlagsForKube122. Since we are adding new flags for 1.26+, I created a new var

Reply:
Got it!